components/Tabs/examples/TabBarIndicatorTemplateExample.swift

```diff
@@ -0,0 +1,119 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import UIKit
+import CoreGraphics
+
+import MaterialComponents.MaterialAppBar
+import MaterialComponents.MaterialPalettes
+import MaterialComponents.MaterialTabs
+
+class TabBarIndicatorTemplateExample: UIViewController {
+
+  // MARK: Properties
+  var alignment: MDCTabBarAlignment {
+    get {
+      return tabBar.alignment
+    }
+    set(newAlignment) {
+      tabBar.setAlignment(newAlignment, animated: true)
+    }
+  }
+
+  lazy var alignmentButton: MDCRaisedButton = self.setupAlignmentButton()
+
+  lazy var appBar: MDCAppBar = self.setupAppBar()
+
+  lazy var tabBar: MDCTabBar = {
+    let tabBar = MDCTabBar()
+    tabBar.alignment = .justified
+
+    tabBar.items = [
+      UITabBarItem(title: "Fly", image: nil, tag:0),
+      UITabBarItem(title: "Sleep", image: nil, tag:0),
```

Comment:
Can we mix all three tab types in the example just to show how the template will automatically change depending on the tab content?
* Text-only
* Image-only
* Text and Image

Reply:
Good idea. Done